stability-checker/before-commit.sh

```diff
@@ -8,21 +8,21 @@ INVERTED='\033[7m'
 NC='\033[0m' # No Color
 
 echo -e "${INVERTED}"
-echo "USER: " + $USER
-echo "PATH: " + $PATH
-echo "GOPATH:" + $GOPATH
+echo "USER: " + "$USER"
```

Comment:
maybe w could change it to something like`echo "USER: ${USER}"` ? A bit simpler I guess